### PR TITLE
Add new characters.

### PR DESCRIPTION
**Pull request by RhodesPeter**
Related #66

Added new characters to project and added to choose-your-avatar page.

### CODE REVIEW

File: public/views/choose-an-avatar.hbs [hunk 1]
```diff
@@ -3,9 +3,9 @@
     <h1 class="avatar__title">Choose your character</h1>
   </div>
   <div class="carousel">
-    <a class="carousel-item" href="#one!"><img src="assets/lion.svg"></a>
-    <a class="carousel-item" href="#two!"><img src="assets/lion-crown.svg"></a>
-    <a class="carousel-item" href="#three!"><img src="assets/lion-hat.svg"></a>
+    <a class="carousel-item" href="#one!"><img src="assets/lion2.svg"></a>
```
Comment (des-des) [inline public/views/choose-an-avatar.hbs]: Has lion1 been deleted? Can we find a better name?

**Comment by RhodesPeter** [comment]
Thanks @des-des. I've renamed lion2.svg to lion.

**Comment by des-des** [comment]
Happy to merge, but tests are failing. Is that okay?

**Comment by RhodesPeter** [comment]
Thanks @des-des, I've fixed the tests.

**Pull request merged**